packages/app/src/App.vue

```diff
@@ -12,22 +12,26 @@
         style="width: 300px"
       >
         <q-card-section>
-          <div style="color:#343434;" class="text-h6">Are you using Chrome?</div>
+          <div
```

Comment:
I'd also recommend moving this dark mode fix to another PR. It'll help keep the PRs more single-featured and it'd help you out with Hacktoberfest :)

Reply:
its here: https://github.com/grey-software/toonin/pull/235